tests/global_communication/test_domain_decomposition.cpp

```diff
@@ -0,0 +1,30 @@
+#include "../gtest.h"
+
+#include <cstdio>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <vector>
+
+#include <communication/communicator.hpp>
+#include <communication/global_policy.hpp>
+
+using namespace nest::mc;
+
+using communicator_type = communication::communicator<communication::global_policy>;
+
+static bool is_dry_run() {
+    return communication::global_policy::kind() ==
+        communication::global_policy_kind::dryrun;
+}
+
+TEST(domain_decomp, basic) {
```

Comment:
Could you give a hint what type of tests you would expect here?
As a source code comment